tests/foreach/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -135,4 +149,13 @@ foreach (
     var_dump($x, $y);
 }
 
+foreach (
+    [
+        'one' => ['string', 'other-string'],
+        'two' => ['string', 'other-string']
+    ]
+    as $key => $aliases
```

Comment:
Without group `$aliases` moved on next line (ugly), but there are a lot of space